tests/python_tests/helpers/perf.py

```diff
@@ -155,7 +155,39 @@ def delete_reports():
         shutil.rmtree(path)
 
 
-def write_to_report(test_config, run_types, results):
+def _flatten(list):
+    return [item for sublist in list for item in sublist]
+
+
+def _dataclass_names(parent, obj):
+    """Provides the **names** of the columns for the report"""
+    return [f"{parent}.{f.name}" for f in fields(obj)]
+
+
+def _dataclass_values(obj):
+    """Provides the **values** of the columns for the report"""
+    return [getattr(obj, f.name) for f in fields(obj)]
+
+
+def report_header(params, result):
+    columns = _flatten(
+        [
+            _dataclass_names(param, value) if is_dataclass(value) else [param]
+            for param, value in params.items()
+        ]
+    )
+
+    for run_type, stats in result.items():
+        for i, _ in enumerate(stats):
+            columns += [
+                f"mean({run_type.name}[{i}])",
+                f"variance({run_type.name}[{i}])",
+            ]
+
+    return columns
+
+
+def write_to_report(test_config, result):
     assert "LLK_HOME" in os.environ, "Environment variable LLK_HOME is not set"
     root = os.environ["LLK_HOME"]
```

Comment:
```suggestion
    llk_home = os.environ.get("LLK_HOME")
    assert llk_home, "Environment variable LLK_HOME is not set"
```
Can use `os.environ.get()` for better clarity and consistent error message control.